CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(ni_grpc_device_server C CXX)
 
+include(CreateVirtualEnvironment.cmake)
```

Comment:
I noticed that you placed this file in the repo top-level. Consider placing it in a `cmake` or subdirectory as Alex did in #343 and adding that directory to the `CMAKE_MODULE_PATH`. Note that you also shouldn't need the `*.cmake` file extension here.